src/import-data/ImportDataDestination.test.ts

```diff
@@ -439,7 +443,7 @@ describe('ImportDataDestination', () => {
       setup({ props });
 
       // Act
-      const newWorkspaceButton = screen.getByText('Start with a new workspace');
+      const newWorkspaceButton = screen.getByText('Create a new workspace');
```

Comment:
I didn't variabilize `Create a new workspace` because its use is spread quite a bit further through the codebase into unrelated modules.  I didn't want to explode the testing scope and so opted to leave it as is.